api/auth.js

```diff
@@ -30,7 +30,7 @@ GET.auth.discord = async ({ url }) => {
   if (!session?.user) return new Response('Bad State', UNAUTHORIZED)
   const authResponse = await fetch(`${DISCORD}/oauth2/token`, {
     method: 'POST',
-    headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+    headers: { 'content-type': 'application/x-www-form-urlencoded' },
```

Comment:
normalized all headers to be lower case, this removed bug of depending on some uppercase headers.